bigtable-client-core-parent/bigtable-client-core/src/main/java/com/google/cloud/bigtable/grpc/RpcMetricsV2.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright 2019 Google LLC.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.cloud.bigtable.grpc;
+
+import com.google.cloud.bigtable.metrics.BigtableClientMetrics;
+import com.google.cloud.bigtable.metrics.Meter;
+import com.google.cloud.bigtable.metrics.Timer;
+
+public class RpcMetricsV2 {
+  private final Timer operationTimer;
+  private final Timer rpcTimer;
+  private final Meter failureMeter;
+
+  public static RpcMetricsV2 createRpcMetrics(String methodName) {
+    String prefix = "grpc.method." + methodName;
+    return new RpcMetricsV2(
+        BigtableClientMetrics.timer(
+            BigtableClientMetrics.MetricLevel.Info, prefix + ".operation.latency"),
+        BigtableClientMetrics.timer(
+            BigtableClientMetrics.MetricLevel.Debug, prefix + ".rpc.latency"),
+        BigtableClientMetrics.meter(BigtableClientMetrics.MetricLevel.Info, prefix + ".failure"));
+  }
+
+  private RpcMetricsV2(Timer operationTimer, Timer rpcTimer, Meter failureCounter) {
+    this.operationTimer = operationTimer;
+    this.rpcTimer = rpcTimer;
+    this.failureMeter = failureCounter;
+  }
+
+  public Timer.Context timeOperation() {
+    return operationTimer.time();
+  }
+
+  public Timer.Context timeRpc() {
+    return rpcTimer.time();
+  }
+
+  public void markFailure() {
+    failureMeter.mark();
+  }
```

Comment:
It's kind of arbitrary which metrics are housed here. Why not add firstResponse.latency as well?